subsys/testsuite/ztest/include/zephyr/ztest_test_new.h

```diff
@@ -53,6 +53,10 @@ struct ztest_suite_stats {
 /**
  * Setup function to run before running this suite
  *
+ * Note: Never zassert in the setup. As this will kill the main test thread causing the test process
+ * to hang. Instead, use zassume (for behavior tested elsewhere, see the zassume function doc) or
```

Comment:
This should probably be some kind of Doxygen link.

Reply:
Removed doc as it's no longer pertinent